deploy/evmos/120_check_EvmosBTCTokens.ts

```diff
@@ -0,0 +1,43 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+import { isTestNetwork } from "../../utils/network"
+import { BigNumber } from "ethers"
+
+const USD_TOKENS_ARGS: { [token: string]: any[] } = {
+  WBTC: ["Wrapped Bitcoin", "WBTC", "8"],
+  RENBTC: ["renBTC", "RENBTC", "8"],
+}
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, getChainId } = hre
+  const { deploy, execute, getOrNull, log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  for (const token in USD_TOKENS_ARGS) {
+    const token_contracts = await getOrNull(token)
+    if (!token_contracts) {
+      await deploy(token, {
+        from: deployer,
+        log: true,
+        contract: "GenericERC20",
+        args: USD_TOKENS_ARGS[token],
+        skipIfAlreadyDeployed: true,
+      })
+      // If it's on hardhat, mint test tokens
+      if (isTestNetwork(await getChainId())) {
```

Comment:
Does `isTestNetwork` apply to testnets like ropsten / koven?

Reply:
we set this manually in utils/networks.ts, so yes